Tools/autotest/arducopter.py

```diff
@@ -1329,15 +1328,15 @@ def fly_ArduCopter(binary, viewerip=None, use_map=False, valgrind=False, gdb=Fal
     return True
 
 
-def fly_CopterAVC(binary, viewerip=None, use_map=False, valgrind=False, gdb=False, frame=None, params=None, gdbserver=False):
+def fly_CopterAVC(binary, viewerip=None, use_map=False, valgrind=False, gdb=False, frame=None, params=None, gdbserver=False, speedup=30):
```

Comment:
Any reason this was changed to a default of 30?

Reply:
I was playing around with mavnative the other night and this crept in...  reverted to 10